libp2p/routing_record.nim

```diff
@@ -127,7 +127,7 @@ proc getSignedPeerRecord*(pb: ProtoBuffer, field: int,
 proc getSignedPeerRecord*(envelopeBuf: seq[byte]): Result[PeerRecord, EnvelopeError] =
   let
     envelope = ? Envelope.decode(envelopeBuf, EnvelopeDomain)
-    spr = ? PeerRecord.decode(envelope.payload).orErr(EnvelopeInvalidProtobuf)
+    spr = ? PeerRecord.decode(envelope.payload).mapErr(x => EnvelopeInvalidProtobuf)
```

Comment:
Whats the difference with `orErr` in this instance? It looks like it's doing exactly the same in a more generic/less explicit way?

Reply:
I would use orErr, but it has only been available for 2 weeks in stew, and I would rather not force libp2p users to update stew (especially for something as easy to work-around)